src/main/java/ch/njol/skript/classes/data/SkriptClasses.java

```diff
@@ -85,6 +87,7 @@ public SkriptClasses() {}
 						"kill the loop-entity")
 				.since("2.0")
 				.after("entitydata", "entitytype", "itemtype")
+				.supplier(() -> new ArrayIterator<>(Classes.getClassInfos().toArray(new ClassInfo[0])))
```

Comment:
```suggestion
				.supplier(() ->Classes.getClassInfos().iterator())
```

Reply:
![image](https://user-images.githubusercontent.com/98935832/189338294-1a9e8227-7f4f-4dcc-90c4-a1bdd0622db8.png)

Reply:
casting it should work fine